piggybank/src/piggybank/Dime.java

```diff
@@ -0,0 +1,43 @@
+package piggybank;
+
+public class Dime extends Money
+{
+    // pass in method from Money
+    public Dime(int count)
+    {
+        super(count);
+    }
+
+    public Dime()
+    {
+        super();
+    }
+
+    // from Money
+    @Override
+    public double getValue()
+    {
+        return count * 0.10;
+    }
```

Comment:
It looks like you meant getTotal here.. This way of getting the total works too instead of setting the initial value in the constructor. however you still need a getValue getter, that only return the value of one coin.